.github/workflows/go.yml

```diff
@@ -54,28 +54,24 @@ jobs:
         include:
           - arch-label: AMD64
             arch: amd64
-            go: 1.17
-            runs-on: ubuntu-latest
-            staticcheck: v0.2.2
+            go: 1.19
+            runs-on: ubuntu-latest            
           - arch-label: AMD64
             arch: amd64
-            go: 1.18
-            runs-on: ubuntu-latest
-            staticcheck: v0.3.3
+            go: '1.20'
+            runs-on: ubuntu-latest            
           - arch-label: ARM64
             arch: arm64v8
-            go: 1.17
-            staticcheck: v0.2.2
+            go: 1.19            
             runs-on: ["self-hosted", "arm", "linux"]
           - arch-label: ARM64
             arch: arm64v8
-            go: 1.18
-            staticcheck: v0.3.3
+            go: '1.20'            
             runs-on: ["self-hosted", "arm", "linux"]
     env:
       ARCH: ${{ matrix.arch }}
       GO: ${{ matrix.go }}
-      STATICCHECK: ${{ matrix.staticcheck }}
+      STATICCHECK: v0.4.5
```

Comment:
I think that we can remove this because `archery docker run` reads `.env` automatically and we have `STATICCHECK=v0.4.5` in `.env`.